url/README.md

```diff
@@ -1,6 +1,21 @@
-The test for browsers is `a-element.html`. The reusable format is `urltestdata.txt`, which
-is not documented in detail. Reverse engineering through `urltestparser.js` should not be
-too hard. Documentation welcome!
+The test for browsers is `a-element.html`. The reusable format is `urltestdata.json`,
```

Comment:
While you're here maybe note that the tests are actually:

- a-element.html
- url-constructor.html
- a-element-xhtml.xhtml (although this seems like I forgot to update it :()